package.json

```diff
@@ -3,16 +3,14 @@
   "version": "1.2.2",
   "private": true,
   "scripts": {
-    "lint": "eslint lib/",
+    "lint": "eslint lib/ ./*.js",
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "mocha --opts test/mocha.opts 'test/**/*.test.js'",
-    "posttest": "npm run lint",
+    "pretest": "npm run lint",
```

Comment:
This is based on the idea of multiple layers of defense (see the slightly outdated https://ariya.io/2012/12/quality-code-via-multiple-layers-of-defense). We ran the simplest and quickest checks first (linting). If nothing fails then we ran the next level (unit tests)... etc.